examples/e-commerce/src/ui.js

```diff
@@ -0,0 +1,37 @@
+const filtersButtons = [
+  ...document.querySelectorAll('[data-action="open-overlay"]'),
+];
+const closeOverlayButtons = [
+  ...document.querySelectorAll('[data-action="close-overlay"]'),
+];
+const resultsContainer = document.querySelector('.container-results');
```

Comment:
FYI, I've move these queries into functions in Angular and Vue to avoid querying before rendering.